Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/wss/CONNECTSignatureProcessor.java

```diff
@@ -88,25 +88,25 @@ void inlineSignatureAttachments(SoapMessage soapMsg, Element signatureElem) thro
             inlineSignatureValueIncludes(signatureElem, attachments);
 
         } catch (IOException ioe) {
-            throw new WSSecurityException(ErrorCode.FAILURE,ioe,"Failed to inline attachments to signature.");
+            throw new WSSecurityException(ErrorCode.FAILURE, ioe, "Failed to inline attachments to signature.");
         }
     }
 
     private void inlineDigestValueIncludes(Element element, Collection<Attachment> attachments) throws IOException {
         NodeList digestNodes = element.getElementsByTagNameNS(SamlConstants.XML_SIGNATURE_NS,
-                SamlConstants.DIGEST_VALUE_TAG);
+            SamlConstants.DIGEST_VALUE_TAG);
 
         inlineIncludes(digestNodes, attachments);
     }
 
     private void inlineSignatureValueIncludes(Element element, Collection<Attachment> attachments) throws IOException {
```

Comment:
The handleMessage() method was inherited from the superclass org.apache.wss4j.dom.processor.SignatureProcessor. The method signature cannot be removed without altering the inheritance contract.